src/js/angular/core/services/repositories.service.js

```diff
@@ -222,7 +222,6 @@ repositories.service('$repositories', ['$http', 'toastr', '$rootScope', '$timeou
             }
         };
 
-        that.getLocations();
```

Comment:
This is probably failing the tests.
A wild guess is that views expect the locations to be fetched repository service when they load.